src/constants/poster.ts

```diff
@@ -0,0 +1 @@
+export const mintDate = '2022-05-23T00:00:00';
```

Comment:
nit: can we rename this to all caps `MINT_DATE` ? it's a small pattern we prefer that makes it easy to identify consts where they're used!

Reply:
Sure! Should consistent with other constant